### PR TITLE
APEXCORE-543 ContainerInfo will contain Map of operators, with Key re…

### PR DESCRIPTION
**Pull request by sandeshh**
…presenting Id and Value representing name.

@davidyan74 please review.

### CODE REVIEW

File: engine/src/main/java/com/datatorrent/stram/webapp/ContainerInfo.java [hunk 1]
```diff
@@ -43,6 +45,7 @@
   public long lastHeartbeat;
   @RecordField(type = "stats")
   public int numOperators;
+  public Map<Integer, String> operatorsIdandName;
```
Comment (davidyan74) [inline engine/src/main/java/com/datatorrent/stram/webapp/ContainerInfo.java]: How about just call this `operators`?

File: engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java [hunk 2]
```diff
@@ -450,6 +451,12 @@ public ContainerInfo getContainerInfo()
     ci.state = container.getState().name();
     ci.jvmName = this.jvmName;
     ci.numOperators = container.getOperators().size();
+    ci.operatorsIdandName = new HashMap<>();
```
Comment (davidyan74) [inline engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java]: Make this a TreeMap so that the output of the json will be sorted.

**Review by davidyan74 — CHANGES_REQUESTED** [review]

File: engine/src/main/java/com/datatorrent/stram/webapp/ContainerInfo.java (hunk 1 above)
Comment (sandeshh) [reply engine/src/main/java/com/datatorrent/stram/webapp/ContainerInfo.java]: Yes, that is better. Done.

File: engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java (hunk 2 above)
Comment (sandeshh) [reply engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java]: Done.

**Pull request merged**